recipes/ammonite-repl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "ammonite-repl" %}
+{% set version = "2.5.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/com-lihaoyi/Ammonite
```

Comment:
We prefer if you would build from release tarballs here instead of cloning the whole sourcetree. Or is there a reason for this?

Reply:
Yes, unfortunately there seems to be a pretty deeply embedded check in the build systems used in many Scala packages that the build is being done in a git repository and it fails with an error if built from the tarball.  I'm guessing it's used at the very least for getting the version, but may also be pulling other information as well. I haven't found any way to disable this and I'm hesitant to try to "hack" things too much here in a way that might break in the future.